fabric-dimensions-v1/src/main/java/net/fabricmc/fabric/api/dimension/v1/EntityPlacer.java

```diff
@@ -23,16 +23,18 @@
 
 /**
  * Responsible for placing an Entity once they have entered a dimension.
- * Stored by a FabricDimensionType, and used in Entity::changeDimension.
+ * used in Entity::changeDimension.
+ *
+ * @deprecated Experimental feature, may be removed or changed without further notice due to potential changes to Dimensions in subsequent versions.
  *
  * @see FabricDimensions
- * @see FabricDimensionType
  */
+@Deprecated
 @FunctionalInterface
 public interface EntityPlacer {
```

Comment:
how about something like `DimensionTeleportHandler` or just `TeleportHandler`? `EntityPlacer` is very vague and it leads to vague class names in user code

Reply:
Right before this place in javadoc word `used` is not capitalized at the start of a sentence

Reply:
This doesnt actually have anything to do with the teleportation. Its just handling the spawn (location, and possibly portal creation) of the entity in the other world.

Reply:
`TeleportListener` or `TeleportPlacer`?

Reply:
For me, `EntityPlacer` can be anything related to entity spawning, entities summoning other entities for attack etc
`TeleportationSomething` is significantly clearer

Reply:
`TeleportListener` is wrong, possibly `TeleportPlacer`?